Classes/Controller/AuthenticationController.php

```diff
@@ -280,8 +280,11 @@ public function startPasswordResetAction($username)
 
         $this->request->setFormat('txt');
         $message->setBody($this->view->render('passwordResetMail'), 'text/plain');
+
         $this->request->setFormat('html');
-        $message->addPart($this->view->render('passwordResetMail'), 'text/html');
+        if ($this->getSettingValue('passwordReset.mail.addHtmlPart')) {
```

Comment:
Can you please also add this to the default settings in `initializeAction()`?

Reply:
Sure, actually now that I see it what is the use case for having the defaults copied there? When people forget to include the TS template? :-) AFAIK most extensions usually just use `$this->settings`.

Reply:
This was started with 4174d3a5c74bd5ac47a3fc5cf5e0d0dd7b65a3e6. Remember what I wrote about having a description is great? ;-) Anyways, the defaults here include values which are reliably set at runtime and yes, covering cases where the static template was forgotten was one of the factors here. This part needs some review in the future.

Reply:
Personally, I think you'd be fine with only having the runtime-value-based defaults there. Might make more sense to warn the user/error out if the static template was forgotten.